core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -412,6 +412,14 @@ package object config {
       .intConf
       .createWithDefault(1)
 
+  private[spark] val STORAGE_DECOMMISSION_ENABLED =
+    ConfigBuilder("spark.storage.decommission.enabled")
+      .booleanConf.createWithDefault(false)
```

Comment:
Please add `version` and split the line like the following.
```scala
private[spark] val STORAGE_DECOMMISSION_ENABLED =
     ConfigBuilder("spark.storage.decommission.enabled")
       .version("3.1.0")
       .booleanConf
       .createWithDefault(false)
```

Reply:
done.